bundles/org.openhab.binding.epsonprojector/src/main/java/org/openhab/binding/epsonprojector/internal/discovery/MulticastListener.java

```diff
@@ -103,47 +107,26 @@ public boolean waitForBeacon() throws IOException {
      * Example Epson beacon:
      * AMXB<-UUID=000048746B33><-SDKClass=VideoProjector><-GUID=EPSON_EMP001><-Revision=1.0.0>
      */
-    private void parseAnnouncementBeacon(DatagramPacket packet) {
+    private @Nullable Map<String, Object> parseAnnouncementBeacon(DatagramPacket packet) {
         String beacon = (new String(packet.getData(), StandardCharsets.UTF_8)).trim();
-
         logger.trace("Multicast listener parsing announcement packet: {}", beacon);
 
-        clearProperties();
+        if (beacon.contains("EPSON") && beacon.contains("VideoProjector")) {
```

Comment:
Did you purposely remove the `toUpperCase` calls? I thought those were there in case a future projector changed the case of those items (which admittedly seems unlikely).

Reply:
This was something that I had on the back burner for a while. There was a problem with another binding where similar code had issues on systems that were using a non-english language locale. It seems a little safer to match the actual value. I might revisit this.

Reply:
Is it ok now? 
Why no more using toUpprrcase for VideoProjector?

Reply:
It is not necessary. The string 'VideoProjector' is the given SDK class name and will always be capital case as defined by the whitepaper I found on the AMX beacon spec. The few other examples that I found for a projector beacon supported this. Here is the whitepaper, the device class names are on the last page: https://trade.amx.com/assets/whitePapers/AMX.Dynamic.Device.Discovery.White.Paper.pdf